backend/CHANGELOG.md

```
@@ -1,5 +1,18 @@
 # Changelog
 
+## 0.9.1 (2025-11-18)
+
+## What's Changed
+* fix: Remove dummy data from explore workflows, workflows ui improvements & minor bug fixes by @aryanranderiya in https://github.com/theexperiencecompany/gaia/pull/328
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)